ocs_ci/deployment/acm.py

```
@@ -188,8 +194,138 @@ def deploy_downstream(self):
+                ):
+                    if ds_data is None:
+                        continue
+                    ds = json.loads(ds_data.stdout)
+                    containers = ds["spec"]["template"]["spec"].get("containers", [])
+                    init_containers = ds["spec"]["template"]["spec"].get(
+                        "initContainers", []
+                    )
+                    all_images = [c["image"] for c in containers + init_containers]
+                    if all(img == routeagent_image for img in all_images):
+                        logger.info(
+                            f"DaemonSet image updated on cluster '{cluster_name}'"
```

Ruff (0.16.2)

[warning] Logging statement uses f-string

(G004)